llvm/test/CodeGen/X86/stack-protector-msvc.ll

```diff
@@ -97,9 +102,13 @@ define void @test_vla(i32 %n) nounwind ssp {
 ; MSVC-X64: callq escape
 ; MSVC-X64: movq [[SLOT]](%rbp), %rcx
 ; MSVC-X64: xorq %rbp, %rcx
-; MSVC-X64: callq __security_check_cookie
+; MSVC-X64:	movq	__security_cookie(%rip), %rax
```

Comment:
Can we avoid loading the cookie into a register?

Reply:
""

Reply:
done